### ISSUE TITLE
I want to re-open and re-purpose this repository

### ISSUE THREAD

**Issue by hyperthunk** [haskell-distributed/cloud-haskell#20]
@facundominguez @qnikst - any objections to this idea (below)?

I am thinking about using this repository to create an integration build, that can be run either in a pipeline (which would mean moving to a different CI provider, since travis doesn't really support build pipelines) or scheduling to run daily.

The idea is that we create an integration testing project, which executes a much broader range of cloud haskell libraries against a much broader set of use cases. A set of `stack-X.yaml` configurations can pull in the very latest of a matrix of our dependencies, and test accordingly. This would allow us to verify multiple configurations of things that are not limited to `distributed-process`, but could also pull in and utilise stuff like `simplelocalnet`, multiple alternative `network-transport` implementations (e.g., 0MQ, named pipes, CCI, etc), exercise some of the _platform libraries_ like `d-p-task`, `d-p-execution`, `d-p-supervisor`, and so on. 

We could also use this capability to exercise tests that run multiple executables.

I will do some further investigation on various CI provider options, and post back my findings.

**Comment by facundominguez** [comment]
Makes sense. But maybe it needs to be renamed to `cloud-haskell-integration-tests` or the like to avoid confusing newcomers.

**Comment by hyperthunk** [comment]
Agreed. Will choose a sensible name once the CI strategy is decided.
